### PR TITLE
Remove all references to lscsoft-glue from hveto

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This PR removes explicit references to `lscsoft-glue` from hveto, and thus removes any explicit dependency on `lal`. This is still brought in from `gwdetchar`, but I'm working on that as well.

### CODE REVIEW

File: bin/hveto [hunk 1]
```diff
@@ -237,7 +236,7 @@ pchannel = cp.get('primary', 'channel')
 
 # read auxiliary cache
 if args.auxiliary_cache:
-    acache = Cache.fromfilenames(args.auxiliary_cache)
+    acache = read_cache(args.auxiliary_cache)
```
Comment (alurban) [inline bin/hveto]: @duncanmmacleod, is this missing an import line, perhaps this one?

```python
from gwpy.io.cache import read_cache
```

File: setup.cfg [hunk 2]
```diff
@@ -44,10 +44,11 @@ install_requires =
 	gwpy >=0.14.0
 	gwtrigfind
 	jinja2
-	lscsoft-glue >=2.0.0
 	lxml
+	MarkupPy
```
Comment (alurban) [inline setup.cfg]: @duncanmmacleod, should this be `MarkupPy >=1.14`?

**Comment by alurban** [comment]
@duncanmmacleod, I've posted a couple of very minor comments, otherwise I think this looks great.

**Comment by duncanmmacleod** [comment]
@alurban, ok, I think this is ready.

**Pull request merged**